Test/CommandLine/Test-JoinCommandLine.ps1

```diff
@@ -0,0 +1,94 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+function Start-TestFixture
```

Comment:
All new tests must use Pester. Blade is obsolete. Convert both new test fixtures to Pester. Here's a [Convert-BladeFixtureToPester script that will do some of the converting](https://bitbucket.org/splatteredbits/blade/src/5aaf45b496b07c358565658beb33fad7fa40f578/Blade/Convert-BladeFixtureToPester.ps1?at=default&fileviewer=file-view-default).